woocommerce/wc-admin
# Migrate Stock Panel to Homescreen

Fixes #5239.

This PR primarily relocates the Stock Panel to the Homescreen.

Several supporting changes were made:

* Updated the `<ProductImage>` styling to match designs (affects Reviews panel as well)
* Added "last order date" to the `/wc-analytics/products` endpoint response
* Let `<AccordionPanel>` prop `initialOpen` toggle the panel if it hasn't been interacted with by the user (allows a panel to be closed while data fetching is occurring, if desired)
* Enhanced the "items" data store to update total counts whenever identical queries (from a totals perspective) are issued - allows for the Panel `count` to change without issuing another query specifically for the total count, since that header will be returned in the query for more items

### Accessibility




### Screenshots

![Screen Shot 2020-11-24 at 9 56 33 AM](https://user-images.githubusercontent.com/63922/100126617-2c445500-2e4c-11eb-9680-1867e11fb710.png)

### Detailed test instructions:

- Ensure your test store has inventory management enabled
- With no products out of stock, verify the panel says "0" and is closed
- With less than 5 products out of stock, verify the panel shows the exact number of placeholders while loading
- With more than 5 products out of stock, verify the panel shows only 5 placeholders while loading
- (Still with 5+) Update a product so it is no longer out of stock
- Verify the product dismisses and another is fetched
- Verify the count decreases
- Update another product
- Click "undo" on the snackbar notice
- Verify the changes were undone
- Verify the stock panel is no longer in the header



### Changelog Note:



Enhancement: move Stock management to the home screen.
